catalog/spatial/wfs/spatial-wfs-featuretransformer-xstream/src/test/java/org/codice/ddf/spatial/ogc/wfs/transformer/xstream/XStreamWfs11FeatureTransformerTest.java

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.ogc.wfs.transformer.xstream;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import ddf.catalog.data.Metacard;
+import java.io.BufferedInputStream;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.Optional;
+import javax.xml.namespace.QName;
+import javax.xml.transform.stream.StreamSource;
+import net.opengis.wfs.v_1_1_0.FeatureTypeType;
+import org.apache.ws.commons.schema.XmlSchema;
+import org.apache.ws.commons.schema.XmlSchemaCollection;
+import org.codice.ddf.spatial.ogc.wfs.catalog.common.FeatureMetacardType;
+import org.codice.ddf.spatial.ogc.wfs.catalog.converter.FeatureConverter;
+import org.codice.ddf.spatial.ogc.wfs.catalog.converter.FeatureConverterFactory;
+import org.codice.ddf.spatial.ogc.wfs.catalog.converter.impl.GenericFeatureConverter;
+import org.codice.ddf.spatial.ogc.wfs.catalog.mapper.MetacardMapper;
+import org.codice.ddf.spatial.ogc.wfs.catalog.metacardtype.registry.WfsMetacardTypeRegistry;
+import org.codice.ddf.spatial.ogc.wfs.featuretransformer.WfsMetadata;
+import org.codice.ddf.spatial.ogc.wfs.v110.catalog.common.Wfs11Constants;
+import org.junit.Before;
+import org.junit.Test;
+
+public class XStreamWfs11FeatureTransformerTest {
+
+  private static final QName PETER_PAN_NAME =
+      new QName("http://www.neverland.org/peter/pan", "PeterPan");
+
+  private static final String MAPPED_ATTRIBUTE = "ext.PeterPan.PANAlias";
+
+  private static final String SOURCE_ID = "testSource";
+
+  private XStreamWfs11FeatureTransformer transformer;
+
+  @Before
+  public void setup() {
+    transformer = new XStreamWfs11FeatureTransformer();
+    transformer.setMetacardTypeRegistry(mockMetacardTypeRegistry());
+  }
+
+  @Test
+  public void testMetacardMappers() {
+    transformer.setMetacardMappers(Collections.singletonList(mockMetacardMapper()));
+
+    InputStream inputStream =
+        new BufferedInputStream(getClass().getResourceAsStream("/FeatureMember.xml"));
```

Comment:
Shouldn't we be closing the `BufferedInputStream` used in these unit tests?

Reply:
Done